entries/i/id.muni.cz.json

```
@@ -0,0 +1,18 @@
+{
+  "MUNI Unified Login": {
```

Entries don't list the authentication systems but the websites using them. Here that would be Mazarykova univerzita or Masaryk University.
```suggestion
  "Mazarykova univerzita": {
```